### PR TITLE
fix bug: cpu stress can not reach pod limit in some situation

### LINKED ISSUE #3100: cpu stress can not reach pod limit in some situation
## Bug Report

**What version of Kubernetes are you using?**

Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.15+k3s1", GitCommit:"60f1e80dcb07a32a42688e149778778ffb00ca9a", GitTreeState:"clean", BuildDate:"2021-01-14T22:50:21Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.15+k3s1", GitCommit:"60f1e80dcb07a32a42688e149778778ffb00ca9a", GitTreeState:"clean", BuildDate:"2021-01-14T22:50:21Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}

**What version of Chaos Mesh are you using?**

2.1.1

**What did you do?**

set cpu stress experiment `cpu 3` and `cpu-load 30`
and the resource limit of target pod  is 500m
 
image

the cpu usage only reach about 300m

**What did you expect to see?**

the cpu usage should reach about 500m

**What did you see instead?**

**Output of chaosctl**

### PR DESCRIPTION
**Pull request by xlgao-zju**
### What problem does this PR solve?

close #3100 

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [x] Manual test (add steps below)

set cpu stress experiment cpu 3 and cpu-load 30 and the resource limit of target pod is 500m

then we got: the cpu usage reaches about 500m

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
add --cpu-load-slice 10 --cpu-method sqrt args for cpu stress by default
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Comment by xlgao-zju** [comment]
ping @STRRL @FingerLeader for review

File: api/v1alpha1/stresschaos_types.go [hunk 1]
```diff
@@ -120,6 +120,7 @@ func (in *Stressors) Normalize() (string, string, error) {
 		}
 	}
 	if in.CPUStressor != nil && in.CPUStressor.Workers != 0 {
+		cpuStressors += " --cpu-load-slice 10 --cpu-method sqrt "
```
Comment (STRRL) [inline api/v1alpha1/stresschaos_types.go:123]: please add comments about why we need these two flags. :D

Comment (STRRL) [inline api/v1alpha1/stresschaos_types.go:123]:
```suggestion
		cpuStressors += " --cpu-load-slice 10 --cpu-method sqrt"
```

the last whitespace seems not required, please remove it.

**Review by STRRL — COMMENTED** [review]
Thanks for your contribution! Basic LGTM! 

Please take a look on the comments

**Comment by xlgao-zju** [comment]
@STRRL updated, PTAL

File: api/v1alpha1/stresschaos_types.go [hunk 2]
```diff
@@ -120,6 +120,10 @@ func (in *Stressors) Normalize() (string, string, error) {
 		}
 	}
 	if in.CPUStressor != nil && in.CPUStressor.Workers != 0 {
+		// Without these to args, we may not reach the resource limit of pod,
+		// especially when we set cpu workers > 1
+		// More details see: https://github.com/chaos-mesh/chaos-mesh/issues/3100
+		cpuStressors += " --cpu-load-slice 10 --cpu-method sqrt"
```
Comment (STRRL) [inline api/v1alpha1/stresschaos_types.go:126]:
```suggestion
		// Without these two args, we may not reach the resource limit of pod,
		// especially when we set cpu workers > 1
		// More details see: https://github.com/chaos-mesh/chaos-mesh/issues/3100
		cpuStressors += " --cpu-load-slice 10 --cpu-method sqrt"
```

Comment (xlgao-zju) [reply api/v1alpha1/stresschaos_types.go]: Done

**Comment by xlgao-zju** [comment]
@STRRL updated

**Review by STRRL — APPROVED** [review]
LGTM! Thanks for your contribution!

**Review by YangKeao — APPROVED** [review]
LGTM

**Comment by STRRL** [comment]
> cherry pick to release-2.1 failed

I would manually cherry pick it.

**Pull request merged**